frontend/pages/home_page.js

```diff
@@ -1,33 +1,75 @@
 import React, { Component, PureComponent } from 'react';
 import { Redirect } from 'react-router';
 import appRoutes from 'src/constants/routes';
+import http from 'src/frontend/services/http';
 
 export default class HomePage extends Component {
   state = {
     loading: false,
     error: '',
     redirectTo: '',
+    products: '',
   }
 
   goToProductNewPage = () => {
     this.setState({ redirectTo: appRoutes.productNew });
   }
 
+  fetchProducts = () => {
+    http.get('/product').then(
```

Comment:
this is not a big deal, but to keep things consistent, I like the syntax of putting params for nested functions in the line with the .then:

```
http.get('/product').then(res => {
  this.setState();
}).catch(e => {
  console.log();
});
```

Reply:
also, probably name the param 'response' or 'res' instead of just 'data'